### DISCUSSION TITLE
Many security issues reported by GCP Container scanning

### DISCUSSION THREAD

**Discussion by cydergoth** [tiangolo/meinheld-gunicorn-flask-docker#162] [Questions] [answered]
470 to be precise, of which 4 critical and 48 high. Most of these can probably be fixed with a later version of the underlying OS base image

**Comment by ambitious-octopus** [comment]
Are there any updates on this issue?

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! I suspect this was solved a long time ago as this image is based in the official Python images and it's deployed automatically. :nerd_face: :rocket:
